http/headers/set-cookie.json

```diff
@@ -507,26 +519,38 @@
                     }
```

Comment:
Above here is edge, which adds support for schemeful behind a preference `#schemeful-same-site` in version 86. Now Chrome 89 removed this flag. Is it possible Edge did too, and this was just omitted?

I looked for the flag in edge:flags and nothing came up on Edge 96.

Reply:
I'm pretty confident this would have been enabled by default in Edge 89 as well, and that the data just wasn't mirrored from Chrome.  Would you be willing to update this while we're here?